contracts/Governance/GovernorBravoDelegate.sol

```diff
@@ -0,0 +1,516 @@
+pragma solidity ^0.5.16;
```

Comment:
How are we going to deal with contracts using different solidity versions?

Reply:
We will create multiple Solidity compilers in the configuration to address the scenario where these lower version contracts are currently not interacting with any higher version contracts.